Automattic/wp-calypso
# chore(framework): Switch to file refs for packages

#### Changes proposed in this Pull Request

* Switch to `file:` refs for internal packages
* Stop using `lerna bootstrap` for `npm ci`
* Ditch the package level locks, they're now part of the main lock
* Update circle and docker builds

#### Testing instructions

* `npm ci` in the main repo should work
* `npm start` should work
* `npm run build-docker` should build a docker instance
* Tests should pass
* Validate that this doesn't bloat the build bundle with duplicated dependencies
